### ISSUE TITLE
"Мирнороли" и негодование, которое часто вызывает данная аномалия.

### ISSUE THREAD

**Issue by Sage4or** [TauCetiStation/Rules#53]
Бывает, что некоторые игроки решают играть по особенному, по мирному, помогая экипажу, решая его проблемы и так далее, тому подобное. Данные случаи вызывают негодования как у гостов, иногда даже у админов, так и у живых космонавтиков, которые находятся в раунде с таким недо-антагом.
![image](https://user-images.githubusercontent.com/59475362/73261240-bac05980-41dc-11ea-951e-9f91ecc0d065.png)

У нас же, по сути, антагонист противостоит каждому члену экипажа, будь то тест сабжект, либо капитан, но "мирнороли" полностью безобразят саму суть антагониста, делая для людей, которые голосовали за ЛЮБОЙ режим, кроме эксты, как раз саму эту эксту, только с щитспавном в виде, например, мага с имбовой хиллкой, который отнимает работу у врачей, а все цели выполнил путем выпрашивания, либо же пока никто не видит

Предлагаю: в правила антагонистов добавить пункт о запрете мирноролей (антагонистов, по типу ниндзи с целями на защиту, трейторов с аналогичными, не учитывать).

Голосовач:
https://forum.taucetistation.org/t/disskusion-l-vout-mirno-roli/15604

**Comment by Motzord** [comment]
Выпилить мага, потому что это единственная мирнороль. (Ну нинжа МБ)

**Comment by Sage4or** [comment]
> Выпилить мага, потому что это единственная мирнороль. (Ну нинжа МБ)

Видел мирно-нюку, мирно-нинжу, мирно-трейторов, мирно-генку, мирно-мага, мирно-культ
Всех не выпилить :с

**Comment by Motzord** [comment]
> > Выпилить мага, потому что это единственная мирнороль. (Ну нинжа МБ)
> 
> Видел мирно-нюку, мирно-нинжу, мирно-трейторов, мирно-генку, мирно-мага, мирно-культ
> Всех не выпилить :с

Мирно нюка это очень редко и смешно (не путать со стелсонюкой)
Мирнонинжа по дефолту появляются ибо нинжи две и у одного задание на выпил другого
Мирно генку тоже видел, вроде лольдем был, забавно но не круто
Мирнокульт не видел. (не путай со стелсокультом)

**Comment by Sage4or** [comment]
> > > Выпилить мага, потому что это единственная мирнороль. (Ну нинжа МБ)
> > 
> > 
> > Видел мирно-нюку, мирно-нинжу, мирно-трейторов, мирно-генку, мирно-мага, мирно-культ
> > Всех не выпилить :с
> 
> Мирно нюка это очень редко и смешно (не путать со стелсонюкой)
> Мирнонинжа по дефолту появляются ибо нинжи две и у одного задание на выпил другого
> Мирно генку тоже видел, вроде лольдем был, забавно но не круто
> Мирнокульт не видел. (не путай со стелсокультом)

Ладно, нинзи не в счет тогда уж, подправлю

Мирнокульт-культ с мирной агитацией как религия и тд, часто бывает, когда цель на 5 аколитов, реже с Насри
Мирногенку тот же Нежлик делал недавно, сдав всех генок СБ и главам. Испортил другим ролям раунд, как то некрута

**Comment by kildjen** [comment]
Да, делал мирногенку. Кому-то зашло, кому-то нет. В любом случае это не превратило раунд в эксту, которую многие так не любят. На СБ там были настолько расторопные ребята, что даже зная имена и имея всю мету не смогли ничего сделать.

А если в общем по теме, то мирнорольки - круто, но только те, которые развлекают экипаж, а не превращают раунд в эксту.

**Comment by Sage4or** [comment]
> Да, делал мирногенку. Кому-то зашло, кому-то нет. В любом случае это не превратило раунд в эксту, которую многие так не любят. На СБ там были настолько расторопные ребята, что даже зная имена и имея всю мету не смогли ничего сделать.
> 
> А если в общем по теме, то мирнорольки - круто, но только те, которые развлекают экипаж, а не превращают раунд в эксту.

Игра про артистов, которые мирно развлекают космонавтиков, точно не про НЁХ и катастрофу на космической станции, да?

Если бы все мирнороли только притворялись мирными, а потом под середину так раунда творили дичь, в которую вовлечена вся станция, не возникло бы этого ишью, но, увы, таких единицы, а зачастую раунд с такими людьми- экста

**Comment by ViDLiathan** [comment]
Предлагаю формулировку в правилах: Роли должны играть в механ и обязаны выполнять цели. Правило о необходимости отыгрыша персонажа не действует в отношении ролей.

**Comment by LudwigVonChesterfield** [comment]
предлагаю сторителлера - у рольки задачи "по времени", если она не выполняет - её пускает в расход система забирая вещи рольки, и даёт кому-то ещё. как только выполнена задача "по времени", появляется новая задача :peka:

**Comment by FatFatFat1** [comment]
Это тупо, когда я тритор я забиваю на гринтекст и иду фанится. Я не мирнороль ,так как конфликтую с персоналом ,но меня твой сторителлер покарает

**Comment by Haggerd** [comment]
> предлагаю сторителлера - у рольки задачи "по времени", если она не выполняет - её пускает в расход система забирая вещи рольки, и даёт кому-то ещё. как только выполнена задача "по времени", появляется новая задача :peka:

Предлагаешь, чтобы рольки упарывались в гринтекст? Игра не про победку, а про то, что игроки сами делают интересные ситуации

**Comment by LudwigVonChesterfield** [comment]
Откуда тогда появляется вот эта проблема?

**Comment by Haggerd** [comment]
> Откуда тогда появляется вот эта проблема?

Из-за игроков. Откуда же ещё. Надо же не призывать игроков к нормальному поведению, а просто выпиливать/ограничивать. Ведь, чем меньше контента и вариативности, тем проще.

**Comment by Sakuya-Izayoi** [comment]
Если вербально мотивировать, то те самые стелсоролеры не обратят внимание. Если банить, то назовут тиранами.

**Comment by WallShrabnic** [comment]
Ha-ha

No
